extensions/amp-audio/0.1/amp-audio.js

```diff
@@ -67,6 +67,36 @@ export class AmpAudio extends AMP.BaseElement {
     this.registerAction('pause', this.pause_.bind(this));
   }
 
+  /** @override */
+  mutatedAttributesCallback(mutations) {
+    const src = mutations['src'];
+    const controlsList = mutations['controlsList'];
+    const loop = mutations['loop'];
+
+    if (src !== undefined || controlsList !== undefined || loop !== undefined) {
```

Comment:
Per our offline discussion, we should not rebuild the audio element, in case the page author is just toggling controlsList, loop. We should only build once and update these properties if they are defined.

Reply:
Did you forget to push some changes here?

Reply:
My bad, my pre-push failed.